WordPress/Docs/PHP/StrictInArrayStandard.xml

```diff
@@ -0,0 +1,45 @@
+<documentation title="Strict In Array Syntax">
+    <standard>
+    <![CDATA[
+    Strict in array syntax must be used.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: strict in array syntax.">
+        <![CDATA[
+in_array( 1, [ '1', 1, true ], <em>true</em> );
+]]>
+        </code>
+        <code title="Invalid: non-strict in array syntax.">
+        <![CDATA[
+in_array( 1, [ '1', 1, true ]<em></em> );
```

Comment:
`em` tags can be removed here - they have no content.

Reply:
I'd actually suggest changing it to have the last whitespace between the tags, so it will highlight where the missing parameter is expected.

```suggestion
in_array( 1, [ '1', 1, true ]<em> </em>);
```